typeDefs.js

```diff
@@ -846,6 +854,10 @@ const typeDefs = gql`
     rate: Int!
   }
   type Mutation {
+    addMaterialsBlock(materialsBlock: MaterialsBlockInput!): MaterialsBlock
```

Comment:
you return just type MaterialsBlock..what's will happen if the method will return an error?

Reply:
you need MaterialsBlock result including error type
also, you need check needed type in resolvers.js. ask @lhalam how he did check for needed type in his own PR

Reply:
Thanks for your comments, fix)